controllers/postController.js

```diff
@@ -184,26 +191,46 @@ const postCount = async (req, res) => {
     return res.status(400).send({ message: "[NotExistedKeywordId] Error occured" });
   }
 
-  const keywordId = req.params.keywordId;
+  let period;
+  if (isValidString(req.query.period)) {
+    if (isEmptyString(req.query.period)) {
+      period = PERIOD.WEEKLY;
+    } else {
+      period = req.query.period;
+    }
+  } else {
+    return res.status(400).send({ message: "[InvalidPeriod] Error occured" });
+  }
```

Comment:
```suggestion
  const period = getPeriod(req.query.period);
  if (period.includes("Invalid")) {
    return res.status(400).send({ message: `[${period}] Error occured` });
  }
```
추후 리팩토링 할 때 별도 모듈로 관리해볼 수 있겠습니다!
잘 쪼개는게 중요할 것 같은데 위 처럼 변경해볼 수 있었어요.
잠깐 예시로 테스트 해봤을 때, 의도대로 동작해서 추후 같이 고민해봐요!

```js
// 예시
// 별도 모듈 파일 정의 ("../utils/getPeriodParams")
const { isValidString, isEmptyString } = require("../utils/validation");
const { PERIOD } = require("../config/constants");

const getPeriod = (period) => {
  if (isValidString(period)) {
    if (isEmptyString(period)) {
      return PERIOD.WEEKLY;
    } else {
      return period;
    }
  } else {
    return "InvalidPeriod";
  }
};

module.exports = {
  getPeriod,
};

```

Reply:
유효성 검사와 에러 메시지에 대한 로직 모듈화는 너무 좋은 아이디어인 것 같아요!
아무래도 모든 함수의 상단에서 유효성 검사 로직이 반복되고 있다보니 모듈화하면 코드가 간결해져서 가독성이 더 좋을 것 같네요!
시연 후 리팩토링 때 꼭 적용해보겠습니다! 👍